### PR TITLE
Organize imports might insert surplus round brackets

### PR DESCRIPTION
**Pull request by mlangc**
This PR fixes a bug in `CommentsUtils.stripComment` that resulted in [Ticket #1002166](https://www.assembla.com/spaces/scala-ide/tickets/1002166-extra-round-brackets-inserted-when-organize-imports-called#/activity/ticket:). Further information can be found in the commit messages.

### CODE REVIEW

File: src/test/scala/scala/tools/refactoring/tests/sourcegen/CommentsUtilsTest.scala [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  }
+
+  @Test
+  def testSplitCommentWithMultilineExamples() {
+    val tripleQuote = "\"\"\""
+    
+    /*
+     * Attention:
+     *  These test cases contain a lot of relevant white space chars; make sure that they are not stripped
+     *  away by safe actions.
```
Comment (sschaef) [inline src/test/scala/scala/tools/refactoring/tests/sourcegen/CommentsUtilsTest.scala]: This is not scalable. Please add a `$` sign (or any other sign) at the end of such lines and strip it away later.

File: src/main/scala/scala/tools/refactoring/implementations/OrganizeImports.scala [hunk 2]
```diff
@@ -69,12 +69,15 @@ abstract class OrganizeImports extends MultiStageRefactoring with TreeFactory wi
...
-        trace("Participant %s:", name)
+        trace(s"$this:")
         trees.foreach(trace("-- %s", _))
         res.foreach(trace("++ %s", _))
       }
     }
+
+    override def toString = s"Participant[$name]"
+
+    private def name = getClass.getSimpleName
```
Comment (sschaef) [inline src/main/scala/scala/tools/refactoring/implementations/OrganizeImports.scala]: `getSimpleName` is not safe on Scala code. Use `getSimpleClassName` from `scala.tools.refactoring` package object instead.

File: src/test/scala/scala/tools/refactoring/tests/sourcegen/CommentsUtilsTest.scala (hunk 1 above)
Comment (mlangc) [inline src/test/scala/scala/tools/refactoring/tests/sourcegen/CommentsUtilsTest.scala]: Right indeed.. I already stumbled over this myself :-/.

**Comment by sschaef** [comment]
LGTM

**Pull request merged**